test/forms/Validators.test.js

```diff
@@ -244,6 +245,100 @@ describe('Validators', () => {
         });
     });
 
+    describe('isPositiveNumber', () => {
+        it('should return true when the value is 100', () => {
+            expect(isPositiveNumber(100)).to.be.true;
+        });
+
+        it('should return true when the value is the string "100"', () => {
+            expect(isPositiveNumber('100')).to.be.true;
+        });
+
+        it('should return true when the value is 1.9', () => {
+            expect(isPositiveNumber(1.9)).to.be.true;
+        });
+
+        it('should return true when the value is the string "1.9"', () => {
+            expect(isPositiveNumber('1.9')).to.be.true;
+        });
+
+        it('should return true when the value is -1.9', () => {
```

Comment:
The description says `true` but the test checks for `false`.
@adhbh Could you please review and make sure that the test descriptions are correct?